qf_lib/data_providers/bloomberg_beap_hapi/bloomberg_beap_hapi_data_provider.py

```diff
@@ -250,22 +250,46 @@ def expiration_date_field_str_map(self, ticker: BloombergTicker = None) -> Dict[
     def supported_ticker_types(self):
         return {BloombergTicker, BloombergFutureTicker}
 
-    def get_tickers_universe(self, universe_ticker: BloombergTicker, date: Optional[datetime] = None) -> List[BloombergTicker]:
+    def get_tickers_universe(self, universe_ticker: BloombergTicker, date: Optional[datetime] = None,
```

Comment:
Is there any point in having this `date` parameter? It's anyway set to today, and if the user sets it to something else it returns an error. 

I would suggest removing the parameter and renaming the function to like `get_current_tickers_universe` or something like that

Reply:
It is a function inherited from the parent class. It is the only class for which date is not supported